packages/grpc-native-core/index.d.ts

```diff
@@ -1189,7 +1199,7 @@ declare module "grpc" {
    * @param error The error, if the call failed
    * @param value The response value, if the call succeeded
    */
-  export type requestCallback = (error: ServiceError | null, value: any) => void;
+  export type requestCallback<T> = (error: ServiceError | null, value: T) => void;
```

Comment:
The `value` argument can be `null` or `undefined` if the `error` argument is non-null.

Reply:
So that should be extended to `T | null`.